netdata-installer.sh

```diff
@@ -136,12 +136,12 @@ ACLK="${ACLK}"
 
```

Comment:
The ideal scenario, everything that is inside `run_stress_test.sh` will come to this file and `packaging/installer/install-required-packages.sh`. I suggest this to be your first priority, because this will allow us to test your PR everywhere.

Reply:
@thiagoftsm I've made some changes that should allow the PR to get built and installed (including Fluent-Bit) just by running `./netdata-installer.sh --enable-logsmanagement`. The only precondition is to manually install `flex` and `bison` packages (I've asked @Ferroin to help with adding these required packages in `install-required-packages.sh` and anywhere else that is necessary). 

Alpine should also work if `fts-dev` gets installed (as `musl` does not provide an `fts.h` interface).